android/app/src/main/assets/capacitor.config.json

```diff
@@ -3,5 +3,11 @@
   "appName": "Lunie",
   "bundledWebRuntime": false,
   "npmClient": "yarn",
-  "webDir": "dist"
+  "webDir": "dist",
+  "plugins": {
+    "IntercomPlugin": {
+      "android-apiKey": "android_sdk-xxxx",
+      "android-appId": "yyyy"
```

Comment:
should be null / empty string? probably doesn't matter.

Reply:
Yep, before app building should be replaced by the correct values by the script Fabo coded yesterday.